<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InputForm</class>
 <widget class="QDialog" name="InputForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Введите данные</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 191);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="labelName">
           <property name="text">
            <string>ФИО:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditName"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="labelMail">
           <property name="text">
            <string>e-mail</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditMail"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_DR">
           <property name="text">
            <string>День рождения</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="dateEdit"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButtonOk">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(255, 255, 255);
}</string>
       </property>
       <property name="text">
        <string>ОК</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="labelN">
         <property name="text">
          <string>№</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditN"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
